roles/ood_user_reg/tasks/main.yml

```diff
@@ -66,3 +66,29 @@
 - name: Build the updated Apache config
   command: /opt/ood/ood-portal-generator/sbin/update_ood_portal
   ignore_errors: yes
+
+- name: Create RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo id -u {{ RegUser_app_user }} &>/dev/null || sudo useradd -c "{{ RegUser_app_user_full_name }}" {{ RegUser_app_user }}'
+  become_user: vagrant
+
+- name : delay
+  command: sleep 2
+
+- name: Change passwd RegUser - {{ RegUser_app_user }} on OHPC
```

Comment:
Do we really need a password?  We really shouldn't set this for a service account.